src/main/resources/camunda/decision_outcome.bpmn

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_1i2ih25" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.21.0" modeler:executionPlatform="Camunda Platform" modeler:executionPlatformVersion="7.15.0">
+  <bpmn:process id="DECISION_OUTCOME" name="Decision outcome" isExecutable="true">
+    <bpmn:callActivity id="StartBussinessDecisionOutcome" name="Start Business Process" calledElement="StartBusinessProcess">
+      <bpmn:extensionElements>
+        <camunda:in variables="all" />
+        <camunda:out variables="all" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0y1b4y2</bpmn:incoming>
+      <bpmn:outgoing>Flow_0agok6u</bpmn:outgoing>
+    </bpmn:callActivity>
+    <bpmn:sequenceFlow id="Flow_0y1b4y2" sourceRef="StartEvent_1" targetRef="StartBussinessDecisionOutcome" />
+    <bpmn:boundaryEvent id="Event_0o3sg3o" name="Abort" attachedToRef="StartBussinessDecisionOutcome">
+      <bpmn:outgoing>Flow_0wksds5</bpmn:outgoing>
+      <bpmn:errorEventDefinition id="ErrorEventDefinition_0uj9pwb" />
+    </bpmn:boundaryEvent>
+    <bpmn:endEvent id="Event_1duzjl5">
+      <bpmn:incoming>Flow_0wksds5</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_0wksds5" sourceRef="Event_0o3sg3o" targetRef="Event_1duzjl5" />
+    <bpmn:sequenceFlow id="Flow_0agok6u" sourceRef="StartBussinessDecisionOutcome" targetRef="CreateDecisionOutcomeDashboard" />
+    <bpmn:callActivity id="EndBussinessDecisionOutcome" name="End Business Process" calledElement="EndBusinessProcess">
+      <bpmn:extensionElements>
+        <camunda:in variables="all" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0buha2s</bpmn:incoming>
+      <bpmn:outgoing>Flow_1ew3g5l</bpmn:outgoing>
+    </bpmn:callActivity>
+    <bpmn:endEvent id="Event_1adcwon">
+      <bpmn:incoming>Flow_1ew3g5l</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_1ew3g5l" sourceRef="EndBussinessDecisionOutcome" targetRef="Event_1adcwon" />
+    <bpmn:startEvent id="StartEvent_1" name="Start">
+      <bpmn:outgoing>Flow_0y1b4y2</bpmn:outgoing>
+      <bpmn:messageEventDefinition id="MessageEventDefinition_11i5zio" messageRef="Message_1fto75j" />
+    </bpmn:startEvent>
+    <bpmn:serviceTask id="CreateDecisionOutcomeDashboard" name="Dashboard Decision Outcome" camunda:type="external" camunda:topic="processCaseEvent">
+      <bpmn:extensionElements>
+        <camunda:inputOutput>
+          <camunda:inputParameter name="caseEvent">CREATE_DASHBOARD_NOTIFICATION_DECISION_OUTCOME</camunda:inputParameter>
```

Comment:
As mentioned in the civil-service PR, I would split this between a claimant & defendant event, so that we can have a caseData.isApplicant1Represented & caseData.isRespondent1Represented check in civil-service to make sure we're dealing with LiP rather than LR.

Reply:
done